### ISSUE TITLE
Use maven-android-sdk-deployer to deploy sdk jar to local maven repo

### ISSUE THREAD

**Issue by ruleant** [gildegoma/chef-android-sdk#14]
As discussed in https://github.com/travis-ci/travis-ci/issues/1395#issuecomment-41911519 : 
Android SDK Jar files > 4.1.1.4 are not available in maven central. A workaround is to install those jar files from the SDK to the local maven repo ~/.m2/repository/

Project [maven android sdk deployer](https://github.com/mosabua/maven-android-sdk-deployer) does exactly this.

Fe. to install android-19 jar files, go up the directory tree : 

```
mvn clean install -N
cd platforms
mvn clean install -N
cd android-19
mvn clean install
```

Maybe @mosabua can shed a light on a way to integrate his tool and use the same component filter used when installing the android components with `android update sdk --filter tools,platform-tools,build-tools-19.0.3,android-19`

**Comment by danielpassos** [comment]
Hi @gildegoma any progress on or in your roadmap still working on it?

**Comment by gildegoma** [comment]
@danielpassos No, no progress. At the moment it is not on my roadmap of personal needs, but I keep the issue open since its value is quite evident to me.

Actually, the relieving step would be to resolve the https://github.com/simpligility/maven-android-sdk-deployer/issues/236 prerequisite. Help is welcome!
